### PR TITLE
CMP-2457: PCI-DSS v4 Requirement 5

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Good part of Requirement 5 is `not applicable`
- While we could add some rules and checks for anti-malware solutions, it will likely not completely satisfy the requirement, and will largely depend on the deployment parameters.

### CODE REVIEW

File: controls/pcidss_4_ocp4.yml [hunk 1]
```diff
@@ -1334,18 +1332,41 @@ controls:
         malware.
       levels:
         - base
-      status: pending
+      status: supported
```
Comment (rhmdnd) [inline controls/pcidss_4_ocp4.yml:1335]: Can you refresh me on the difference between `supported` and `partial` in this context?

**Review by rhmdnd — COMMENTED** [review]
/lgtm

Just one question inline.

File: controls/pcidss_4_ocp4.yml (hunk 1 above)
Comment (jan-cerny) [reply controls/pcidss_4_ocp4.yml:1335]: https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html#reporting-status

partial: While work has been done to address this control, there is still work needed to fully address it.

supported: The control is addressed by the product (but is missing content automation).

**Comment by yuumasato** [comment]
Just rebased to address the product stability test.

**Comment by yuumasato** [comment]
Rebased on latest master

**Review by rhmdnd — APPROVED** [review]
/lgtm

**Pull request merged**